scripts/run_e2e_tests.sh

```diff
@@ -86,6 +86,9 @@ fi
 set -e
 source $(dirname $0)/setup.sh || exit 1
 source $(dirname $0)/setup_gae.sh || exit 1
+if [ "$TRAVIS" = true ]; then
```

Comment:
Shouldn't this be `==`? Also, I think we can drop `""` over the environment variables -- both should work I think.

Reply:
Jeez this is a bad one. Thanks for picking this up. See note below at .travis.yml and let me know what you think!

Reply:
Can you take a look at this? I simply saw it over setup.sh and incorporate over it here.
https://unix.stackexchange.com/questions/16109/bash-double-equals-vs-eq

Reply:
Oh, so `"$TRAVIS" == 'true'` is equivalent to `"$TRAVIS" = true`? If this is the case, prefer the former just to maintain consistency.

Reply:
So do you want me to change the if statement over at setup.sh too?

Reply:
If there are any differing occurrences, it would be great if we make them consistent to this pattern too! So, go ahead and change them if there are any.

Reply:
I made the changes! Anything else I should take care of?